hepstats/hypotests/fitutils/utils.py

```diff
@@ -46,3 +47,15 @@ def array2dataset(dataset_cls, obs, array, weights=None):
         return dataset_cls.from_numpy(obs=obs, array=array, weights=weights)
     else:
         return dataset_cls(obs=obs, array=array, weights=weights)
+
+
+def get_nevents(dataset):
+    """ Returns the number of events in the dataset """
+
+    nevents = dataset.nevents
+
+    if not isinstance(nevents, numbers.Number) and "zfit" in str(dataset):
```

Comment:
This part special for zfit is very strange. It cannot be if the package is a general API for others too :S.

Reply:
The problem is that `dataset.nevents` in `zfit` returns a tensor that has to be evaluated. Maybe there is a a way to get directly a Number in the `zfit` API that I am not aware of, @mayou36 ?

Reply:
Hmm, so this is a place where cross-talk is important. There should be a general method to get just a number, yes.

Reply:
Ok we can solve this later. I will merge the PR.